features/teacher_searches_groups_and_previews_instructional_materials.feature

```diff
@@ -156,32 +156,6 @@ Feature: Teacher can search instructional materials grouped by material type, so
 
 
   @javascript
-  Scenario: Teacher can sort search investigations on the basis of popularity
-    When the Investigation "differential calculus" is assigned to the class "Physics"
-    And the Investigation "differential calculus" is assigned to the class "Geography"
-    And the Investigation "differential calculus" is assigned to the class "Mathematics"
-    And the Investigation "integral calculus" is assigned to the class "Mathematics"
-    And the Investigation "integral calculus" is assigned to the class "Geography"
-    And I follow "Popularity" in Sort By on the search instructional materials page
-    And I should wait 2 seconds
-    And I search for "calculus" on the search instructional materials page
-    Then "differential calculus" should appear before "integral calculus"
-
-
-  @javascript
-  @wip
-  Scenario: Teacher can sort search activities on the basis of popularity
-    When the Activity "intersecting lines" is assigned to the class "Physics"
-    And the Activity "intersecting lines" is assigned to the class "Geography"
-    And the Activity "intersecting lines" is assigned to the class "Mathematics"
-    And the Activity "parallel lines" is assigned to the class "Mathematics"
-    And the Activity "parallel lines" is assigned to the class "Geography"
-    And I follow "Popularity" in Sort By on the search instructional materials page
-    And I should wait 2 seconds
-    Then "intersecting lines" should appear before "parallel lines"
-
-
-  @javascript
```

Comment:
@pjanik We have a story for removing the popularity filter here https://www.pivotaltracker.com/story/show/91504150.